src/ctl/ctl.c

```diff
@@ -0,0 +1,590 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
it's usually first copyright, then a license txt, then SPDX tag, e.g.

```
/*
 *
 * Copyright (C) 2023-2024 Intel Corporation
 *
 * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 *
 */
```

Reply:
![image](https://github.com/user-attachments/assets/0af2e8e3-3c03-4d46-bf1f-9db1526b5224)

Reply:
> ![image](https://private-user-images.githubusercontent.com/7746856/395137439-0af2e8e3-3c03-4d46-bf1f-9db1526b5224.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQwMDM4MjMsIm5iZiI6MTczNDAwMzUyMywicGF0aCI6Ii83NzQ2ODU2LzM5NTEzNzQzOS0wYWYyZThlMy0zYzAzLTRkNDYtYmYxZi05ZGIxNTI2YjUyMjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTJUMTEzODQzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTI2ZWFmYTJlZGE5YmI2ZThlYjY2MzhhNjQ0MjMyYzBhMTk1YmFmNmQwMTkzZGEyMTI2ZmZlZDIyYTgzMDkyYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.87zWqP9bSirND6eCUG81ps5CKlLZRnXIiGEj4qIcoKo)

? 😉

Reply:
> ? 😉

I cannot reply in thread related with that you see
https://github.com/oneapi-src/unified-memory-framework/pull/913#discussion_r1881706417

Reply:
I'm not sure what do you mean? the issue here is about copyright and license - please fix - now we I can see 2 licenses and 2 copyrights in this file